File: Everything/ClientApp/src/models/lifting/LiftSet.ts
export default class LiftSet {
    id?: number = 0;
    liftId?: number | null = null;
    date: Date | string = new Date();
    number: number | null = null;
    reps: number | null = null;
    weight: number | null = null;

    constructor(init?: Partial<LiftSet>) {
        Object.assign(this, init);
    }
}

File: Everything/Models/Lifting/LiftSet.cs
using System;

namespace everything.Models
{
    public class LiftSet
    {
        public int Id { get; set; }
        public int? Number { get; set; }
        public int? Reps { get; set; }
        public int? Weight { get; set; }

        public int LiftSetLinkId { get; set; }
        public virtual LiftSetLink LiftSetLink { get; set; }
    }
}
